inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sortstandalone/SortSourceGroupInfo.java

```diff
@@ -29,8 +29,8 @@
 @Data
 public class SortSourceGroupInfo {
     private static final Logger LOGGER = LoggerFactory.getLogger(SortSourceGroupInfo.class);
-    private static final String KEY_SECOND_CLUSTER_TAG = "second_cluster_tag";
-    private static final String KEY_SECOND_TOPIC = "second_topic";
+    private static final String KEY_BACK_UP_CLUSTER_TAG = "back_up_cluster_tag";
+    private static final String KEY_BACK_UP_TOPIC = "back_up_topic";
```

Comment:
`backup` is one word, please change `back_up` to `backup`, thanks.

Reply:
fixed, thx